src/components/ThemeProvider/tests/ThemeProvider.test.tsx

```diff
@@ -1,31 +1,14 @@
 import React, {useContext} from 'react';
-import {createMount} from 'test-utilities';
+import {mountWithApp} from 'test-utilities/react-testing';
```

Comment:
The `test-utilities` index reexports everything from `test-utilities/react-testing` for the shortest naming

```suggestion
import {mountWithApp} from 'test-utilities';
```